.github/workflows/pr-tests-frontend.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
       - name: Docker on MacOS
         if: steps.changes.outputs.frontend == 'true' && matrix.os == 'macos-latest'
-        uses: crazy-max/ghaction-setup-docker@v1.1.0
+        uses: crazy-max/ghaction-setup-docker@v1.3.0
```

Comment:
Suggest to set `SIGN_QEMU_BINARY=1` env var per https://github.com/crazy-max/ghaction-setup-docker#macos-qemustderr-qemu-system-x86_64-unknown-error

```yaml
    - name: Docker on MacOS
      if: steps.changes.outputs.frontend == 'true' && matrix.os == 'macos-latest'
      uses: crazy-max/ghaction-setup-docker@v1.3.0
      env:
        SIGN_QEMU_BINARY: 1
```